imgix/__init__.py

```diff
@@ -1,4 +1,25 @@
 # -*- coding: utf-8 -*-
+"""
+imgix
+~~~~~
+
+A Python client library for generating URLs with Imgix. Basic usage:
```

Comment:
"imgix" should always be written without capitalization. Will you update all instances in these three files?

Reply:
Will do.